corehq/apps/data_interfaces/tasks.py

```diff
@@ -74,7 +74,7 @@ def run_case_update_rules(now=None):
 
 @serial_task(
     '{domain}',
-    timeout=24 * 60 * 60,
+    timeout=36 * 60 * 60,
```

Comment:
A bit confused by this. This is the lock timeout so are you saying that this task can run for longer than 24 hours?